helm/values.yaml

```diff
@@ -618,6 +618,7 @@ gateway:
   enabled: true
   name: gateway
   type: mongodb
+  dataPlaneId: default
```

Comment:
Wondering if we have to introduce a variable gateway.dataPlaneId in the values.yaml 🤔